test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/PravegaControllerK8sService.java

```diff
@@ -92,12 +92,12 @@ public boolean isRunning() {
                         .thenCompose(o -> {
                            Map<String, Object> spec = (Map<String, Object>) (((Map<String, Object>) o).get("spec"));
                            Map<String, Object> pravegaSpec = (Map<String, Object>) spec.get("pravega");
-                           Map<String, Object> bookkeeperSpec = (Map<String, Object>) spec.get("bookkeeper");
+                           //Map<String, Object> bookkeeperSpec = (Map<String, Object>) spec.get("bookkeeper");
```

Comment:
Is this change related to the change in `CUSTOM_RESOURCE_VERSION_PRAVEGA`? If so, why do we keep it as a comment instead of removing it altogether?

Reply:
System test failed in current release (both p-operator and pravega) because of looking for bookkeeper in specification,  but as per current release bookeeper has separated from PravegaCluster, So removed bookkeeperSpec

Yes, I will remove this line

Reply:
This changes not related to PR so revert the changes